libs/distributed_lcos/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2019-2020 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+# Default location is $HPX_ROOT/libs/distributed_lcos/include
+set(distributed_lcos_headers
+  hpx/distributed_lcos/base_lco.hpp
+  hpx/distributed_lcos/base_lco_with_value.hpp
+  hpx/distributed_lcos/channel.hpp
+  hpx/distributed_lcos/detail/promise_base.hpp
+  hpx/distributed_lcos/detail/promise_lco.hpp
+  hpx/distributed_lcos/lcos_fwd.hpp
+  hpx/distributed_lcos/object_semaphore.hpp
+  hpx/distributed_lcos/packaged_action.hpp
+  hpx/distributed_lcos/promise.hpp
+  hpx/distributed_lcos/server/channel.hpp
+  hpx/distributed_lcos/server/object_semaphore.hpp
+  )
+
+# Default location is $HPX_ROOT/libs/distributed_lcos/include_compatibility
+set(distributed_lcos_compat_headers
+  hpx/lcos/base_lco.hpp
+  hpx/lcos/base_lco_with_value.hpp
+  hpx/lcos/channel.hpp
+  hpx/lcos/detail/promise_base.hpp
+  hpx/lcos/detail/promise_lco.hpp
+  hpx/lcos_fwd.hpp
+  hpx/lcos/object_semaphore.hpp
+  hpx/lcos/packaged_action.hpp
+  hpx/lcos/promise.hpp
+  hpx/lcos/server/channel.hpp
+  hpx/lcos/server/object_semaphore.hpp
+  )
+
+set(distributed_lcos_sources
+  base_lco.cpp
+  base_lco_with_value.cpp
+  base_lco_with_value_1.cpp
+  base_lco_with_value_2.cpp
+  base_lco_with_value_3.cpp
+  promise.cpp
+  )
+
+include(HPX_AddModule)
+add_hpx_module(distributed_lcos
+  COMPATIBILITY_HEADERS OFF
```

Comment:
```suggestion
  COMPATIBILITY_HEADERS ON
```

Reply:
May I change the default in the python script ? I always forget about those..

Reply:
Of course. It's supposed to help us do the right thing after all.